include/FtyPolicies.hpp

```diff
@@ -6,26 +10,28 @@
 
 namespace fty {
 struct AsLowercase {
-  std::string apply(const std::string &String) {
+  static auto apply(const std::string& String) -> std::string {
     std::string ConvertedString(String.size(), '\0');
-    std::transform(String.begin(), String.end(), ConvertedString.begin(),
-                   [](unsigned char Char) { return std::tolower(Char); });
+    std::transform(String.begin(), String.end(), ConvertedString.begin(), [](unsigned char Char) {
+      return std::tolower(Char);
+    });
     return ConvertedString;
   }
 };
 
 struct AsUppercase {
-  std::string apply(const std::string &String) {
+  static auto apply(const std::string& String) -> std::string {
     std::string ConvertedString(String.size(), '\0');
-    std::transform(String.begin(), String.end(), ConvertedString.begin(),
-                   [](unsigned char Char) { return std::toupper(Char); });
+    std::transform(String.begin(), String.end(), ConvertedString.begin(), [](unsigned char Char) {
+      return std::toupper(Char);
+    });
     return ConvertedString;
   }
 };
 
 struct AsOriginal {
-  std::string apply(const std::string &String) { return String; }
+  static auto apply(const std::string& String) -> std::string { return String; }
```

Comment:
Why replace the return type with auto and then give it a trailing return type here?